### PR TITLE
Add mandatory

### PR DESCRIPTION
**Pull request by MiyamotoAkira**
6 sections on the form are now mandatory.

### CODE REVIEW

File: _includes/apply_form.html [hunk 1]
```diff
@@ -10,18 +10,18 @@ <h4 class="modal-title" id="myModalLabel4">{{ include.title }}</h4>
                     <div id="{{ include.target }}-fields">
                         <div class="row">
                             <div class="col-md-6 margin-bottom-20">
-                                <label>First Name</label><input name="{{ include.target }}-firstname" class="form-control" type="text" />
+                                <label>First Name (*)</label><input name="{{ include.target }}-firstname" class="form-control" type="text" />
```
Comment (robertfirek) [inline _includes/apply_form.html:13]: You can add required option to the field (e.g. `<input type="text" name="..." required>`)

File: _includes/apply_form.html [hunk 2]
```diff
@@ -51,21 +56,22 @@ <h4 class="modal-title" id="myModalLabel4">{{ include.title }}</h4>
                         </div>
                         <div class ="row">
                             <div class="col-md-12 margin-bottom-20">
-                                <label>What does Software Craftmanship mean to you?</label>
+                                <label>What does Software Craftmanship mean to you? (*)</label>
                                 <textarea name="{{ include.target }}-craftmanship" class="form-control" rows="10" id="craftmanship"></textarea>
```
Comment (robertfirek) [inline _includes/apply_form.html:60]: Text area can also be `required`

File: _includes/apply_form.html (hunk 1 above)
Comment (MiyamotoAkira) [reply _includes/apply_form.html:13]: On this case we are not using required as we are really not submitting the form, but calling a js method that does an Ajax call.

File: _includes/apply_form.html (hunk 2 above)
Comment (MiyamotoAkira) [reply _includes/apply_form.html:60]: Same as above:
On this case we are not using required as we are really not submitting the form, but calling a js method that does an Ajax call.

**Pull request merged**